sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md

```diff
@@ -37,21 +37,37 @@ if the kube-apiserver address of M Cluster is accessible on any node of the H Cl
       clusterRole: host
     ```
 
-  If you already have a standalone KubeSphere installed, you can change it to a host or member cluster by editing the cluster configuration and wait for a while.
+  If you already have a standalone KubeSphere installed, you can change it to a host cluster by editing the cluster configuration and wait for a while.
 
     ```shell
     kubectl edit cc ks-installer -n kubesphere-system
     ```
 
 ### 2.2. <a name='MemberCluster'></a>Install Member Cluster
 
-* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as the following and wait for the installation to be completed.
+* In order for host cluster to manager member cluster, we need to make the jwtSecret same between them. So first get it from the host by the following command.
+
+    ```bash
+    $ kubectl -n kubesphere-system get cm kubesphere-config -o yaml | grep -v "apiVersion" | grep jwtSecret
+      jwtSecret: "gfIwilcc0WjNGKJ5DLeksf2JKfcLgTZU"
+    ```
+
+* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as the following, input the jwtSecret we have from the step above and wait for the installation to be completed.
```

Comment:
`input the jwtSecret we have from the step above` => `input the corresponding jwtSecret shown above`